packages/dev/s2-docs/pages/s2/home/Home.tsx

```diff
@@ -242,17 +204,33 @@ export function Home() {
         </div>
       </nav>
       <header aria-labelledby={headingId} className={style({marginX: 'auto', paddingX: {default: 16, sm: 40}, paddingY: 96, maxWidth: 1024})}>
-        <h1 id={headingId} className={style({font: 'heading-3xl', marginY: 0, color: 'white'})}>
-          <span className={swapWrapper} >Build apps </span>
+        <h1
+          id={headingId}
+          className={style({
+            font: 'heading-3xl',
+            '--headingFontSize': {
```

Comment:
don't have direct access to fontSize and can't use it as the type because fontSize may expand to multiple values `({--fs, fontSize})`